rust-toolchain.toml

```diff
@@ -1,3 +1,8 @@
 [toolchain]
-channel = "stable"
-targets = ["x86_64-unknown-linux-gnu", "x86_64-unknown-linux-musl", "aarch64-unknown-linux-gnu", "aarch64-unknown-linux-musl"]
+channel = "nightly"
```

Comment:
Is there something you added that requires the nightly channel? I would prefer to stay on stable for now

Reply:
i guess